ol7/templates/oval_5.11_templates/template_OVAL_service_disabled

```diff
@@ -0,0 +1,53 @@
+<def-group>
+  <definition class="compliance" id="service_%SERVICENAME%_disabled" version="1">
+    <metadata>
+      <title>Service %SERVICENAME% Disabled</title>
+      <affected family="unix">
+        <platform>Oracle Linux 7</platform>
+      </affected>
+      <description>The %SERVICENAME% service should be disabled if possible.</description>
+    </metadata>
+    <criteria comment="package %PACKAGENAME% removed or service %SERVICENAME% is not configured to start" operator="OR">
+      <extend_definition comment="%PACKAGENAME% removed" definition_ref="package_%PACKAGENAME%_removed" />
+      <criteria operator="AND" comment="service %SERVICENAME% is not configured to start">
+        <criterion comment="%SERVICENAME% not wanted by multi-user.target" test_ref="test_%SERVICENAME%_not_wanted_by_multi_user_target" />
+        <criterion comment="%SERVICENAME% socket not wanted by multi-user.target" test_ref="test_%SERVICENAME%_socket_not_wanted_by_multi_user_target" />
+        <criterion comment="%SERVICENAME% is not running" test_ref="test_service_not_running_%SERVICENAME%" />
+      </criteria>
+    </criteria>
+  </definition>
+
+  <linux:systemdunitdependency_test check="all" check_existence="any_exist" comment="systemd test" id="test_%SERVICENAME%_not_wanted_by_multi_user_target" version="1">
+    <linux:object object_ref="object_multi_user_target_for_%SERVICENAME%_disabled" />
+    <linux:state state_ref="state_systemd_%SERVICENAME%_off"/>
```

Comment:
Because OL7 uses systemd, could the shared checks be used here as well? They look to be the same as RHEL -- which means we could gain efficiency by using one template.

Reply:
OL manages services exactly in the same manner as RHEL does. As future improvement this templated could be moved to shared templates folder to be utilized at least by ol7 and rhel7 products.